packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -30,7 +31,12 @@ export const CMContext = React.createContext<ContextualMenuContext>({
 export const ContextualMenu = compose<ContextualMenuType>({
   displayName: contextualMenuName,
   usePrepareProps: (userProps: ContextualMenuProps, useStyling: IUseComposeStyling<ContextualMenuType>) => {
-    const { setShowMenu, ...rest } = userProps;
+    const {
+      setShowMenu,
+      shouldFocusOnMount = true,
```

Comment:
Neat syntax, if I understand correctly the point is "use shouldFocusOnMount if provided in userProps, otherwise default to shouldFocusOnMount=True" ?